### ISSUE TITLE
React "material-ui" : unresolved reference React

### ISSUE THREAD

**Issue by up-to-you** [Kotlin/ts2kt#71]
After generation kotlin stubs for react material-ui (ts2kt v0.0.20, material-ui v0.20), runtime results:
```
src/material/index.__MaterialUI.kt:339:33: error: unresolved reference: React
external interface PaperProps : React.HTMLAttributes<Any>, React.Props<Paper> {
                                ^
```

**Comment by bashor** [comment]
RIght now you should convert react declarations by your self and maybe fix some references. Sorry for the inconvenience.

**Comment by up-to-you** [comment]
Can you provide example at least for this case ? I mean React declaration and its attributes.
Thanks in advance!

**Comment by bashor** [comment]
Declarations that are you need are here https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/v15/index.d.ts

**Comment by lamba92** [comment]
yeah but i don't think those are compatible with JetBrains React Wrapper... Is there any solution to make material-ui work with the official wrappers?
